evm.md

```diff
@@ -2153,6 +2232,44 @@ There are several helpers for calculating gas (most of them also specified in th
 
     rule [Cmem]: Cmem(SCHED, N) => (N *Int Gmemory < SCHED >) +Int ((N *Int N) /Int Gquadcoeff < SCHED >)
 
+    rule [Caddraccess]:
```

Comment:
Formatting in this section can be tidied up a bit. Use one line when possible (I recommend disabling line-wrapping in your editor).

Also, line up side-conditions where it makes sense, for example, instead of:

```
requires Ghasaccesslist << SCHED >>
requires notBool Ghasaccesslist << SCHED >>
```

Do:

```
requires         Ghasaccesslist << SCHED >>
requires notBool Ghasaccesslist << SCHED >>
```

Also, you may need to add these new rule names (eg. `EVM.Caddraccess`) to the various `concrete-rules.txt` files present in `tests/specs/...` to get the proofs to pass.